php/PostTypes/BlockPost.php

```diff
@@ -47,7 +47,7 @@ public function __construct() {
 	 */
 	public function register_hooks() {
 		add_action( 'init', [ $this, 'register_post_type' ] );
-		add_action( 'admin_init', [ $this, 'add_caps' ] );
+		add_action( 'plugins_loaded', [ $this, 'add_caps' ] );
```

Comment:
Maybe `admin_init` was a little too late to add the capabilities. It looks by moving it earlier to `plugins_loaded`, the capability is there in time for the submenus to be there.